plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/MavenCoordinates.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Describes a Maven artifact.
```

Comment:
This needs a comment calling out that `equals()` is different.

Reply:
Isn't the comment on the equals method sufficient?

Reply:
Nobody looks at javadoc on `equals()`.

Reply:
OK, done.